code/modules/awaymissions/capture_the_flag.dm

```diff
@@ -280,7 +280,8 @@
 /obj/machinery/capture_the_flag/proc/spawn_team_member(client/new_team_member)
 	var/mob/living/carbon/human/M = new/mob/living/carbon/human(get_turf(src))
 	new_team_member.prefs.copy_to(M)
-	M.set_species(/datum/species/human)
+	if(isplasmaman(M))
```

Comment:
I feel like it'd be better to just future-proof this by checking if they're nonhuman.

Reply:
Point of this was to allow non humans into CTF minus plasmamen (who can't survive in CTF)